### DISCUSSION TITLE
QoL, UI and bugfixes

### DISCUSSION THREAD

**Discussion by snouz** [Albeoris/Memoria#479] [Releases]
[COMPLETE CHANGELOG HERE](https://github.com/Albeoris/Memoria/wiki/Changelog-v2024.05.19)

## Features
- Turn-based mode improvements
- Option for more columns for items/ability
- Fast menu navigation and save/load
- Controller captured when game is unfocused
- Hold RB/Shift + Confirm for fast dialogs
- Holding Confirm works in battle
- Launcher: UKR translation (@DragonOfTheEastUA)

## Fixes
- Fixes bugs: music, cloud save, text, UI and settings
- Backgrounds: Fixed 45 layer orders, 12 smooth movements, 1 animation, 2 border lines, 2 camera bugs
- Lots of backend/modding updates and fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Albeoris/Memoria/releases/tag/v2024.05.19'>QoL, UI and bugfixes</a>.</em>

**Comment by dennissuarezz** [comment]
Hello! Id like to point out a problem I am having with Memoria: The final cutscene doesnt play, it just stays in black screen. It is right after Zidane takes his coat off and says "Bring my beloved dagger to me!". I posted it on reddit and it seems I am not the only one with the problem. (I tried with the latest version and the one before that, both of them dont work)

Link to my post: https://www.reddit.com/r/FinalFantasyIX/comments/1d0k2p3/black_screen_instead_of_ending_cutscene_problem/

Hopefully it can be fixed! Id love to see the entire ending sequence after the final boss fight with the Moguri video enhancement.

Thank you for your work

**Reply by Teamtorrentscripts** [reply]
It's strange that this happens to you, a couple of days ago I was able to see the final scene several times without problems even with MoguriMod installed. Did you try deactivating it in that scene?

**Reply by snouz** [reply]
Hi @dennissuarezz I created a ticket from your report and answered : )

https://github.com/Albeoris/Memoria/issues/500
